string robot_name

---
bool success
string error

geometry_msgs/Pose pose
int32 status

